### PR TITLE
feat(adapter-ui): scheduler status panel on the Evolution page (Spec 55 §7)

### PR DESCRIPTION
**Pull request by laofahai**
## What

Add a read-only **scheduler status panel** to the Evolution admin page (`/admin/evolution`) so an operator can see the autonomous cadence loop's heartbeat at a glance — completing the observability story started by the backend endpoint in #508.

## Panel

Polls `GET /api/evolution/scheduler-status` and renders:
- a status pill: **Running** / **Idle** / **Disabled** / **Unauthorized** / **Unavailable**
- humanized interval, ticks completed/started, last tick time + duration
- an amber **error-streak** row (`lastError` + consecutive count) when the cadence is stuck failing — the exact "stuck on a repeating error" signal #508 added to the backend

It auto-polls on a configurable interval and offers a manual refresh. Strictly read-only — it never triggers a mutation.

## Client

`lib/evolution-api.ts` — `fetchSchedulerStatus()` + a self-contained `SchedulerStatus` wire type (the UI never imports the server/core runtime). Returns a discriminated result (`ok` / `denied` / `error`) so each outcome renders distinctly instead of throwing. `fetchImpl` injection keeps tests free of global fetch mocks.

## Robustness (codex iterative review)

- **Out-of-order responses** — a monotonic request-id guard: only the most recently *started* request commits, so a slow poll can't overwrite a newer heartbeat.
- **Latency > poll interval** — the poll *skips a tick while a request is in flight* (`inFlightRef`), so sustained latency can't make overlapping polls supersede each other forever and leave the panel stale.
- **Duration formatting** — `humanizeMs` rounds to whole seconds before decomposing, so boundary values never render impossible labels like `59m 60s` / `60s` (regression-tested).

## Tests

Pure helpers (ms humanizer incl. boundary carries, response→view reducer, timestamp formatter) and the client (200 configured/unconfigured, field defaults, 401/403→denied, 503/non-2xx/transport/invalid-JSON→error) are unit-tested with `bun:test` (no DOM — this repo's UI tests are logic-only). Full `bun run test` green; typecheck + biome clean; codex clean after the hardening rounds.

Depends on #508 (the endpoint), already merged to main.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: addons/adapter-ui/cap-adapter-ui/src/lib/evolution-api.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+// ── Auth header helper (reuse from api.ts / proposal-api.ts pattern) ──────
+
+function getAuthHeaders(): Record<string, string> {
+  const token = localStorage.getItem("linchkit:token");
+  if (token) {
+    return { Authorization: `Bearer ${token}` };
+  }
+  return {};
+}
```
[the remark this answers is not in the document]
Comment (laofahai) [reply addons/adapter-ui/cap-adapter-ui/src/lib/evolution-api.ts:24]: Good defensive point — added a guard in 09d11f34: `getAuthHeaders()` now returns empty headers when `typeof localStorage === "undefined"` (SSR / router loaders / non-DOM test runners) instead of assuming a browser global. The token path is unchanged in the browser.
